analytics.go

```diff
@@ -387,7 +387,7 @@ func (c *client) getMarshalled(msgs []message) ([]byte, error) {
 }
 
 // Send batch request.
-func (c *client) send(msgs []message) {
+func (c *client) send(msgs []message, count int) {
```

Comment:
can we add some test cases covering this scenario?

Reply:
Can we add test case later? This is a very urgent fix & the customer is waiting for it.